src/java.base/share/classes/sun/security/util/KnownOIDs.java

```diff
@@ -313,6 +313,11 @@ public enum KnownOIDs {
     SHA512withECDSA("1.2.840.10045.4.3.4"),
     SpecifiedSHA2withECDSA("1.2.840.10045.4.3"),
 
+    // PQC
+    ML_KEM_512("1.3.6.1.4.1.22554.5.6.1", "ML-KEM-512"),
+    ML_KEM_768("1.3.6.1.4.1.22554.5.6.2", "ML-KEM-768"),
+    ML_KEM_1024("1.3.6.1.4.1.22554.5.6.3", "ML-KEM-1024"),
```

Comment:
Where are these OIDs from? It seems that NIST has different OIDs assigned to ML-KEM: https://csrc.nist.gov/projects/computer-security-objects-register/algorithm-registration#KEM

Reply:
Fixed.